js/display.js

```diff
@@ -1493,6 +1545,29 @@ function displaySpellDamage(parent_elem, _overallparent_elem, stats, spell, spel
                     }
                 }
             }
+			/*
+					add_summary("Lifesteal Per Hit: ", Math.round(ls_val), "Health");
+					add_summary("Manasteal Per Hit: ", ms_val, "Mana");
+   			*/
+
+			let atkSpdMap = new Map();
+			atkSpdMap.set("SUPER_SLOW", 0.51);
+			atkSpdMap.set("VERY_SLOW", 0.83);
+			atkSpdMap.set("SLOW", 1.5);
+			atkSpdMap.set("NORMAL", 2.05);
+			atkSpdMap.set("FAST", 2.5);
+			atkSpdMap.set("VERY_FAST", 3.1);
+			atkSpdMap.set("SUPER_FAST", 4.3);
+			
+			let curFromMap = atkSpdMap.get(stats.get("atkSpd"));
+			let ls_val = (stats.get("ls") / 3) / curFromMap;
+			let ms_val = (stats.get("ms") / 3) / curFromMap;
+			
+			if(spellIdx === 0) {
+				_damage_display("Health per Hit: ", ls_val, [0, 0, 0, 0, 0, 0], [0, 0, 0, 0, 0, 0]);
```

Comment:
lol this looks jank but if it works it works

maybe the helper function needs to be renamed